CHANGELOG.md

```diff
@@ -1,10 +1,48 @@
 # Changelog
 
+## v0.32.2
+
+*July 31, 2019*
+
```

Comment:
High level summary of what this release is about? Not to duplicate what's actually in the changelog, but just a high level description.